### ISSUE TITLE
Improve API documentation on Bfilter

### ISSUE THREAD

**Issue by buniello** [opentargets/issues#2906]
Following up from this recent [community post](https://community.opentargets.org/t/bfilter-doest-work/990), we realised we need to improve the API documentation.
Discussed at the team leads meeting today.

**Comment by mbdebian** [comment]
@remo87, @prashantuniyal02 , has this been addressed?

**Comment by prashantuniyal02** [comment]
No, I think we need to prioritise this at some point to update the documentation.

**Comment by remo87** [comment]
@prashantuniyal02 Is this being prioritised for the next released? I'm adding @mbdebian to follow up on this.

**Comment by prashantuniyal02** [comment]
To be addressed in API documentation
